rasa/nlu/utils/spacy_utils.py

```diff
@@ -234,10 +234,10 @@ def train_chunk(
         See parent class for more information.
         """
         attribute_docs = self._docs_for_training_examples(
-            training_data_chunk.training_examples
+            training_data.training_examples
```

Comment:
@dakshvar22 it seems to me that these `docs` already contain dense features. I think it should be `train_chunk` then?

Reply:
This one is also like the above point made about `HFTransformers`.